base/applications/notepad/lang/pt-PT.rc

```diff
@@ -103,7 +103,7 @@ BEGIN
     PUSHBUTTON "Ajuda", IDHELP, 8, 170, 50, 14
     DEFPUSHBUTTON "OK", IDOK, 198, 170, 50, 14, BS_PUSHBUTTON
     PUSHBUTTON "Cancelar", IDCANCEL, 254, 170, 50, 14
-    PUSHBUTTON "P&rinter...", psh3, 310, 170, 50, 14
+    PUSHBUTTON "I&mpressora...", psh3, 310, 170, 50, 14
```

Comment:
`LTEXT "&Rodapé:", 0x142, 8, 149, 40, 15`
in this dialog does still collide with 
`AUTORADIOBUTTON "&Retrato", rad1, 16, 80, 52, 12, BS_AUTORADIOBUTTON`
I do suggest that you do fix that as well.